subscribers:
  anymal_state:
    topic: /state_estimator/anymal_state
    queue_size: 10
    latch: false
  joy:
    topic: /joy_manager/joy
    queue_size: 10
  command_velocity:
    topic: /twist_mux/twist
    queue_size: 10
    latch: false
  command_pose:
    topic: /joy_manager/pose
    queue_size: 10
    latch: false
  actuator_states:
    topic: /sensors/actuator_states
    queue_size: 10
  actuator_readings:
    topic: /state_estimator/actuator_readings
    queue_size: 10
    latch: false
  soft_emcy_stop:
    topic: /soft_emcy_stop
    queue_size: 10
  hard_emcy_stop:
    topic: /hard_emcy_stop
    queue_size: 10
  localization:
    topic: /localization_manager/map_in_odom
    queue_size: 1
  imu:
    topic: /state_estimator/imu
    queue_size: 10
    latch: false
